### ISSUE TITLE
Backtesting with a lot of trades, dry run shows no trade?

### ISSUE THREAD

**Issue by PoCk3T** [iterativv/NostalgiaForInfinity#140]
Hi everyone and first of all, awesome work

**Context:** I've been running NFI in dry-run for a while (5 days)
**Problem:** even if the UI shows that the bot is running, with new logs created every few minutes, no trade was ever done
**Troubleshooting done:** I've setup a backtest with the exact same configuration (static pair whitelist, same blacklist, same config-private.json), and the list of trades from the backtesting shows 500+ trades, **including trades done during the time I've been doing dry run** ; additionally, there is no trace of errors in the dry run logs.

Is there anything else I would have missed? :( 

Thank you to any one who would be able to share some feedbacks or tips on what else I could look at / troubleshoot :)

Below is the docker-compose.yml I use, the commands I use to start dryrun or backtesting are as simple as it can get: _docker-compose up backtesting_ and _docker-compose up dryrun_

`---
version: '3'
services:
  backtesting:
    build:
       context: .
       dockerfile: "./docker/Dockerfile.custom"
    container_name: freqtrade-backtesting
    volumes:
      - "./user_data:/freqtrade/user_data"
      - "./NostalgiaForInfinityX.py:/freqtrade/NostalgiaForInfinityX.py"
    command: >
      backtesting
      --strategy-list NostalgiaForInfinityX
      --timerange ${TIMERANGE:-20210101-20211231}
      --config user_data/data/blacklist-binance.json
      --config user_data/data/Binance-USDT-All-Tradable-StaticPairList.json
      --config user_data/data/config-private.json
      --max-open-trades ${MAX_OPEN_TRADES:-5}
      --stake-amount ${STAKE_AMOUNT:-unlimited}
      --export trades
      --export-filename=user_data/backtest_results/backtest_trades_list_export.json

  dryrun:
    build:
      context: .
      dockerfile: "./docker/Dockerfile.custom"
    container_name: freqtrade-dryrun
    ports:
    - "8081:8080"
    volumes:
    - "./user_data:/freqtrade/user_data"
    - "./NostalgiaForInfinityX.py:/freqtrade/NostalgiaForInfinityX.py"
    command: >
      trade
      --strategy NostalgiaForInfinityX
      --dry-run
      --config user_data/data/blacklist-binance.json
      --config user_data/data/Binance-USDT-All-Tradable-StaticPairList.json
      --config user_data/data/config-private-live.json`

**Comment by PoCk3T** [comment]
Thanks a lot @b-zig  for the feedback, I will give it a try.
I'm still curious though, wouldn't dryrun & backtest supposed to behave the same when running over the same period?

**Comment by vallemar** [comment]
@PoCk3T  I am in the same situation as you were, it does not make any purchase, but it seems to start well, the telegram bot works and the dashboard too.
All you did was add @blckwhtx  config and use pairlist-volume-binance-usdt.json?
It doesn't work for me with this config.

Thanks!

**Comment by PoCk3T** [comment]
@vallemar , that's correct, but really start from scratch again, in terms of config files: as in, go with the template and just add whatever else is strictly needed for Freqtrade to run without complaining. 

For non-Americans who can understand a joke not carrying any actual harm, Freqtrade tends to be like a woman, very sensitive, only go with the minimal working config.

If you keep struggling, let me know and I can share my detailed working conf.

**Comment by mbsfernando** [comment]
@PoCk3T  I'm facing the same as you were. would you able to share working config.json for dry run mode ?
